content/rdi/reference/data-transformation-block-types/map.md

```diff
@@ -5,19 +5,19 @@ description: Map a record into a new output based on expressions
 weight: 10
 alwaysopen: false
 categories: ["redis-di"]
-aliases: 
+aliases:
 ---
 
 Map a record into a new output based on expressions
 
 **Properties**
 
-| Name                          | Type     | Description                                   | Required |
-| ----------------------------- | -------- | --------------------------------------------- | -------- |
-| [**expression**](#expression) | `object` | Expression<br/>                               | yes      |
-| **language**                  | `string` | Language<br/>Enum: `"jmespath"`, `"sql"`<br/> | yes      |
+| Name                          | Type               | Description                                   | Required        |
+| ----------------------------- | ------------------ | --------------------------------------------- | --------------- | --- |
+| [**expression**](#expression) | `object`, `string` |                                               | Expression<br/> | yes |
```

Comment:
@spicy-sauce Looks like we have an extra column space here?

Reply:
good catch, the table is corrupted, I'll fix, thank you.